addons/web/static/tests/setup.js

```diff
@@ -17,6 +18,8 @@ export async function setupTests() {
 
   QUnit.testStart(() => {
     odoo = makeTestOdoo();
+    const originalLocale = Settings.defaultLocale;
+    Settings.defaultLocale = "en";
```

Comment:
nitpicking: I would directly do `luxon.Settings.defaultLocale = "en";`, s.t. it is clear that we change the luxon internals (when you see `Settings`, you don't necessarily know what it is, so you have to look at the imports)